plataform_front_end/src/components/loader/loader.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./loader.css";
+
+const max = (n) => n;
+const Loading = (props) => {
+    return <div className="homer"><p>Homer</p></div>
+}
+
+export { Loading, max }
+
+
+export default Loading;
```

Comment:
```suggestion
import React from "react";

import "./loader.css";

const Loading = () => {
    return <div className="homer"><p>Homer</p></div>
}

export default Loading;

```

It looks like this `max` function is not being used, so I'd remove it.